app/models/post.rb

```diff
@@ -1,17 +1,15 @@
-class Post < ActiveRecord::Base
-  belongs_to :author, class_name: 'User'
-  has_many :comments, foreign_key: 'post_id'
-  has_many :likes, foreign_key: 'post_id'
+class Post < ApplicationRecord
+  validates :title, presence: true, length: { maximum: 250 }
 
-  def five_recent_comment
-    posts.limit(5).order(comments(:desc))
-  end
-
-  after_save :update_post_counter
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id', counter_cache: true
+  has_many :likes
+  has_many :comments
```

Comment:
- [ ] As requested in the project requirements, kindly add a validation for comments counter and likes counter in the post model